build.gradle.kts

```diff
@@ -86,13 +86,13 @@ subprojects {
     val compileKotlin by tasks.getting(KotlinCompile::class) {
       kotlinOptions {
         jvmTarget = "11"
-        allWarningsAsErrors = true
+        allWarningsAsErrors = false
```

Comment:
Is this is the default let's just remove the line

Reply:
🤷 I like the explicit call-out that this is _not_ the setting we want, since it's been turned on before, but is actively harmful for maintenance of the project going forward (because it prevents deprecation).